src/CalcManager/UnitConverter.h

```diff
@@ -50,10 +50,6 @@ namespace UnitConversionManager
             accessibleName = nameValue1 + L" " + nameValue2;
         }
 
-        virtual ~Unit()
```

Comment:
It doesn't appear that anything is supposed to inherit from `Unit` so there is no need to have a virtual destructor which adds overhead to this type by giving it a vtable.